src/Components/Web.JS/src/Platform/Circuits/DefaultReconnectDisplay.ts

```diff
@@ -98,16 +113,38 @@ export class DefaultReconnectDisplay implements ReconnectDisplay {
     this.button.style.display = 'block';
     this.reloadParagraph.style.display = 'none';
     this.loader.style.display = 'none';
-    this.message.innerHTML = 'Reconnection failed. Try <a href>reloading</a> the page if you\'re unable to reconnect.';
-    this.message.querySelector('a')!.addEventListener('click', () => location.reload());
+
+    const errorDescription = this.document.createElement('span');
+    errorDescription.textContent = 'Reconnection failed. Try ';
```

Comment:
Strictly speaking this could be simplified. You don't need to create a `span` - you could use `document.createTextNode` and put in the text directly. Then it would be identical to the DOM structure we had before, and `DefaultReconnectDisplay.test.ts` would not need to be changed.

Reply:
Thanks, updated!